src/System.Net.Requests/tests/DummyWebProxy.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Net.Test.Common
```

Comment:
This should not be the namespace for this class since this file is not in the Common\src directory.  It should be the same namespace as the other test files in this folder.